### ISSUE TITLE
Question: logstash 1.5.0 RC2 and logstash.log is growing fast

### ISSUE THREAD

**Issue by remotesyssupport** [elastic/logstash#3101]
In my logstash [1.5.0RC2] setup, I have no STDOUT output defined in my configuration. But I see that the logstash.log file is growing very fast into multiple GB size. 

Previously till 1.5.0.beta1, the log file "logstash.log" would only be populated when there was a stdout output defined.

Is this feature or a bug? If feature, is there a way to reduce it.

**Comment by ph** [comment]
What is the content of the logstash.log, complete events, warning or error?

**Comment by remotesyssupport** [comment]
So I checked it and seems the levelname is either warn or error at present

**Comment by suyograo** [comment]
@remotesyssupport can you provide sample output in your logstash.log?

**Comment by frconil** [comment]
Hi,

I think I have a similar issue. I disabled logging to file and in result logstash.log was under control but now logstash.stdout is getting a copy of each output that is sent to elasticsearch.

I've disabled every stdout option aside from the init.d script one (ie "> ${NAME}.stdout" )
I can try to scrub some of our logs for you if that helps

**Comment by erik-stephens** [comment]
I'm using latest 1.5 (1.5.4) official logstash docker container.  No real progress on #1569 makes this even more painful to digest.  The document fails to index, in my case, because of mapping issues with doc values.  Here is the output when trying to index directly using curl:
```
{"error":"RemoteTransportException[[Spike Freeman][inet[/10.128.213.65:9300]][indices:data/write/index]]; nested: IllegalArgumentException[cannot change DocValues type from SORTED_NUMERIC to SORTED_SET for field \"httpversion\"]; ","status":400}
```

This is what I'm seeing in logstash output for a single document:
```
{:timestamp=>"2015-10-17T00:35:56.175000+0000", :message=>"failed action with response of 500, dropping action: [\"index\", {:_id=>\"4f26c2be818b4127e64db0bf7b49e08bbb42dd36\", :_index=>\"logstash-2015.10.11\", :_type=>\"www-foo-api\", :_routing=>nil}, #<LogStash::Event:0x31f250c @metadata_accessors=#<LogStash::Util::Accessors:0xe7ebd3d @store={\"id\"=>\"4f26c2be818b4127e64db0bf7b49e08bbb42dd36\", \"retry_count\"=>0}, @lut={\"[id]\"=>[{\"id\"=>\"4f26c2be818b4127e64db0bf7b49e08bbb42dd36\", \"retry_count\"=>0}, \"id\"]}>, @cancelled=false, @data={\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, @metadata={\"id\"=>\"4f26c2be818b4127e64db0bf7b49e08bbb42dd36\", \"retry_count\"=>0}, @accessors=#<LogStash::Util::Accessors:0x340908b9 @store={\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, @lut={\"file\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"file\"], \"host\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"host\"], \"offset\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"offset\"], \"type\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"type\"], \"[type]\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"type\"], \"[process]\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"process\"], \"[message]\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"message\"], \"message\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"message\"], \"clientip\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"clientip\"], \"ident\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"ident\"], \"auth\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"auth\"], \"timestamp\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"timestamp\"], \"verb\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"verb\"], \"request\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"request\"], \"httpversion\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"httpversion\"], \"response\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"response\"], \"referrer\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"referrer\"], \"agent\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"agent\"], \"@timestamp\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"@timestamp\"], \"user\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"user\"], \"path\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"path\"], \"remote_ip\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"remote_ip\"], \"agent_name\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"agent_name\"], \"agent_os\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"agent_os\"], \"agent_os_name\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"agent_os_name\"], \"agent_device\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"agent_device\"], \"agent_major\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"agent_major\"], \"agent_minor\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"agent_minor\"], \"[hostname]\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"hostname\"], \"[pop]\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"pop\"], \"hostname\"=>[{\"message\"=>\"1.2.3.4 GET /api/firewall/www1\", \"@version\"=>\"1\", \"@timestamp\"=>\"2015-10-11T18:40:01.000Z\", \"type\"=>\"www-foo-api\", \"auth\"=>\"-\", \"verb\"=>\"GET\", \"httpversion\"=>\"1.1\", \"response\"=>\"304\", \"agent\"=>\"Python-urllib/2.7\", \"user\"=>\"-\", \"path\"=>\"/api/firewall/www1\", \"remote_ip\"=>\"1.2.3.4\", \"agent_name\"=>\"python\", \"agent_os\"=>\"Other\", \"agent_os_name\"=>\"Other\", \"agent_device\"=>\"Other\", \"agent_major\"=>\"2\", \"agent_minor\"=>\"7\"}, \"hostname\"]}>>]", :level=>:warn}
```

**Comment by untergeek** [comment]
Considering this issue resolved due to #4428
